Description: remove interactive choice at launch
 it is no longer useful nowadays
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cool_mzx/cool_mzx.c
+++ b/cool_mzx/cool_mzx.c
@@ -149,31 +149,8 @@
 /*      Alustaa musiikkisysteemin, kyselee laitteet yms.        */
 int mzx_init(void)
 {
-	char	pag[100];
-
-	puts("Select your sound device:");
-	puts("0 - None");
-	puts("1 - 8-bit 22kHz");
-	puts("2 - 16-bit 44kHz");
-	scanf("%s",pag);
-		
-	switch(pag[0])
-	{
-		case '0': laite=0;
-			  sd_ns_init(0,0,0);
-			  break;
-		case '1':
-			laite=1;
-			if(sd_dsp16_init(22050,0,0)==SD_ERR)
-			return(0); 
-			  break;
-		case '2': laite=2;
-			  if(sd_dsp16_init(44100,0,0)==SD_ERR)
-				return(0); 
-			  break;
-		default: return(0);
-	}
-
+	laite=2;
+	if(sd_dsp16_init(44100,0,0)==SD_ERR)
         callbackcount = 0;
         sd_init();
 	sd_setmainvolume(64); 
